packages/db/src/utils.ts

```diff
@@ -1,8 +1,12 @@
-export { defineDbIntegration } from './core/utils.js';
+import type { AstroIntegration } from 'astro';
 import { tableSchema } from './core/schemas.js';
```

Comment:
This is still importing from `core/schemas.js` which imports `core/utils.js` which imports vite.

The rule is that `runtime` should not import inside in `core`. I don't know if this problem is solvable in this way.

Reply:
`core/schemas` no longer imports Vite — I removed that.

Reply:
Yeah, I see that now, but this could easily be broken by a new PR that does need something else from utils. We try to stick with the rule that runtime code only imports runtime code and not core, to prevent mistakes.